src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -762,8 +765,8 @@ class PumpEquipmentDescription:
     esp_parameters = attrib_enum(default=constants.EspParameters.UserDefined)
 
     esp_viscosity_model = attrib_enum(default=constants.PumpViscosityModel.NoModel)
-    esp_thermal_efficiency_model = attrib_enum(
-        default=constants.PumpThermalEfficiencyModel.Constant
+    enable_density_correction: bool = attr.ib(
```

Comment:
"enable" here is a verb, this reads better:

```suggestion
    density_correction_enabled: bool = attr.ib(
```